extras/indexing/src/test/java/mvm/rya/indexing/external/PcjIntegrationTestingUtil.java

```diff
@@ -25,22 +25,6 @@
 import java.util.HashSet;
```

Comment:
another try to avoid making non-meaningful changes comment -- makes it hard to review